.env.example

```
@@ -0,0 +1,8 @@
+OPENAI_API_KEY="XXXXX"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)